mono/utils/mono-error.c

```diff
@@ -18,6 +18,12 @@
 #include <mono/metadata/debug-helpers.h>
 #include <mono/metadata/object-internals.h>
 
+//void error_init(MonoError *error)
```

Comment:
Leftover?

Reply:
Sort of. This is still slightly work in progress. One of the experiments is to leave all the error_inits in and make it a function so I can add/remove all with one file recompiled and relinked. So experimental, and maybe somewhat long term, but can be restored when actually experimenting. Let's see if I can wrap it up though..